src/MathNodeSpec.js

```diff
@@ -27,7 +27,19 @@ const MathNodeSpec: NodeSpec = {
     {tag: 'span[data-latex]', getAttrs},
   ],
   toDOM(node) {
-    return ['span', node.attrs];
+    // Normally, the DOM structure of the math node is rendered by
+    // `MathNodeView`. This method is only called when user selects a
+    // math node and copies it, which triggers the "serialize to HTML" flow that
```

Comment:
do we need to do the same in any other `NodeSpec` files, e.g. `ImageNodeSpec`? https://github.com/chanzuckerberg/czi-prosemirror/blob/master/src/ImageNodeSpec.js#L83

Reply:
Nope.

`ImageNodeSpec` only expects`<img src="..." />` and it does output the right HTML structure.